iotedgedev/modules.py

```diff
@@ -15,6 +14,51 @@ def __init__(self, envvars, utility, output, dock):
         self.dock = dock
         self.dock.init_registry()
 
+    def create(self, name, lang):
```

Comment:
how about change input name "lang" to "type"? the value specify on module type not language. (Consider csharp custom module and csharp function module)